### PR TITLE
Revert Credit Card Telemtry Changes

### PR DESCRIPTION
**Pull request by rocketsroger**
This is to pave the way for Credit Card Telemetry rework with a clean slate

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: app/src/main/java/org/mozilla/fenix/utils/Settings.kt [hunk 1]
```diff
@@ -1038,61 +1038,37 @@ class Settings(private val appContext: Context) : PreferencesHolder {
     )
 
     /**
-     * Stores the number of installed add-ons for telemetry purposes
+     * Storing number of installed add-ons for telemetry purposes
```
Comment (gabrielluong) [inline app/src/main/java/org/mozilla/fenix/utils/Settings.kt:1041]: I am not quite sure about the tense between "Storing" over "Stores". I personally prefer using "Stores" here.

**Review by gabrielluong — APPROVED** [review]

File: app/src/main/java/org/mozilla/fenix/utils/Settings.kt (hunk 1 above)
Comment (rocketsroger) [reply app/src/main/java/org/mozilla/fenix/utils/Settings.kt:1041]: hmm, not sure why this is part of the revert.  I'll double check

Comment (rocketsroger) [reply app/src/main/java/org/mozilla/fenix/utils/Settings.kt:1041]: Confirmed that this is part of the credit card telemetry refactor change.  Just to ensure that I am reverting everything, I'll keep this as part of the revert.

**Comment by mcarare** [comment]
I am a little confused. There already is a PR waiting for review about adjusting the CC telemetry: https://github.com/mozilla-mobile/fenix/pull/20156.
Is this revert needed? Are there other changes planned for CC telemetry? Thanks!

**Comment by rocketsroger** [comment]
> I am a little confused. There already is a PR waiting for review about adjusting the CC telemetry: #20156.
> Is this revert needed? Are there other changes planned for CC telemetry? Thanks!

Oh no, I didn't notice that you were actively working on this.  Yes, I am currently working with Data Science and Glean team to rework credit card telemetry as an example implementation.  There are new requirement for credit card telemetry from the Data Science team.  https://docs.google.com/spreadsheets/d/11G9TNTebUwdDrhuqZYA4uwitV3LyzeLqRsxCj2Fsdrw/edit?skip_itp2_check=true&pli=1#gid=0

This is my mistake of not making a public announcement in slack but only mentioned it in weekly meetings.  Unfortunately your PR needs to be closed.  I will make sure any improvement you made in your PR is carried over to my change.

**Pull request merged**